packages/plugins/typescript/graphql-apollo/src/visitor.ts

```diff
@@ -32,6 +32,7 @@ export class GraphQLApolloVisitor extends ClientSideBaseVisitor<
     const typeImport = this.config.useTypeImports ? 'import type' : 'import';
 
     this._additionalImports.push(`${typeImport} { ApolloClient } from '@apollo/client';`);
+    this._additionalImports.push(`${typeImport} * as Apollo from '@apollo/client';`);
```

Comment:
I'm not convinced this is the best way to update the imports, but... maybe it's good enough? 😁

Reply:
actually let's not do that, I'll change it, the reason is it'll import everything and unless you have treeshaking (which is already tricky), you'll get everything, which we don't want.